lib/pco_api/people/person.ex

```diff
@@ -20,4 +19,15 @@ defmodule PcoApi.People.Person do
   linked_association :phone_numbers
   linked_association :school
   linked_association :social_profiles
+
+  def list, do: get("people/v2/people")
+  def list(params) when is_list(params), do: get(params, "people/v2/people")
```

Comment:
I know you do this in Actions already, but it seems like it might be nicer if `list/1` was just `def list, do: list([])` in this module to save you from needing the endpoint written twice here, makes it clear that they both end up at `get/2`. Not a big deal though.